README.md

```diff
@@ -93,6 +93,7 @@ The global configuration file is located at `.config/kubespider.yaml`, which is
 |  Configuration Item  | Required |                   Function |
 | :-------- | -----: | -------------------: |
 | `proxy` |      No | The request proxy, such as `http://192.168.1.5:1087` |
+| `auth_token`| No | The token for authentication |
```

Comment:
Authentication token, used to protect download requests from being triggered. If public access is required, it can be configured to increase security.